### DISCUSSION TITLE
Getting validation error BR-OPT-00002-0052 when sending customization id with two digit after the dot, for ex "eforms-sdk-1.10"

### DISCUSSION THREAD

**Discussion by jlouvrier** [OP-TED/eForms-SDK#928] [Q&A] [answered]
There seems to be a problem with the rule checking the pattern for the customization ID

When sending <cbc:CustomizationID>eforms-sdk-1.10</cbc:CustomizationID>, we are getting the following error:

<svrl:failed-assert id="BR-OPT-00002-0052" location="/can:ContractAwardNotice/cbc:CustomizationID" test="matches(normalize-space(.),'^eforms-sdk-\d\.\d$')" role="ERROR">
        <svrl:text>OPT-002-notice must match the following pattern: CustomizationID</svrl:text>
    </svrl:failed-assert>

When sending <cbc:CustomizationID>eforms-sdk-1.9</cbc:CustomizationID> , no error.

**Comment by YvesJo** [comment, marked as answer]
Hi,
Please make sure the correct set of schematron files is used for validation or correct parameters are submitted
In SDK 1.9.2, we have:
`	<rule context="/*/cbc:CustomizationID">
		<assert id="BR-OPT-00002-0052" role="ERROR" test="matches(normalize-space(.),'^eforms-sdk-\d\.\d$')">rule|text|BR-OPT-00002-0052</assert>
	</rule>`

while since SDK 1.10.0 we have:
`	<rule context="/*/cbc:CustomizationID">
		<assert id="BR-OPT-00002-0052" role="ERROR" test="matches(normalize-space(.),'^eforms-sdk-\d+\.\d+$')">rule|text|BR-OPT-00002-0052</assert>
	</rule>
`
